torch_geometric/data/batch.py

```diff
@@ -78,10 +81,29 @@ class Batch(metaclass=DynamicInheritance):
     Furthermore, :meth:`~Data.__cat_dim__` defines in which dimension graph
     tensors of the same attribute should be concatenated together.
     """
+    @classmethod
+    def from_batch_index(cls, batch_idx: Tensor) -> Self:
```

Comment:
What is the purpose of this method? This doesn't necessarily look particular useful to me.